dashboard/src/components/authentication/login/login.tsx

```diff
@@ -0,0 +1,149 @@
+import React, { useState, useEffect } from 'react';
+import TextField from '@material-ui/core/TextField';
+import { createStyles, makeStyles, Theme } from '@material-ui/core/styles';
+import Card from '@material-ui/core/Card';
+import CardContent from '@material-ui/core/CardContent';
+import CardActions from '@material-ui/core/CardActions';
+import Button from '@material-ui/core/Button';
+import CardHeader from '@material-ui/core/CardHeader';
+import Link from '@material-ui/core/Link';
+import Navbar from '../../common/navbar/navbar';
+import Footer from '../../common/footer/footer';
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    container: {
+      display: 'flex',
+      flexWrap: 'wrap',
+      width: 400,
+      margin: `${theme.spacing(0)} auto`
+    },
+    loginBtn: {
+      marginTop: theme.spacing(2),
+      flexGrow: 1,
+      background: '#24292e',
+      color: '#fff'
+    },
+    header: {
+      textAlign: 'center',
+      background: '#24292e',
+      color: '#fff'
+    },
+    card: {
+      marginTop: theme.spacing(10)
+    },
+
+  }),
+);
+
+const Login = () => {
+  const classes = useStyles();
+  const [username, setUsername] = useState('');
+  const [password, setPassword] = useState('');
+  const [isButtonDisabled, setIsButtonDisabled] = useState(true);
+  const [helperText, setHelperText] = useState('');
+  const [error, setError] = useState(false);
+
+  const navigation = {
+    brand: { name: 'Environment Platform', to: '/' },
+    links: [
```

Comment:
Why do we need these kind of links

Reply:
This is for the navigation bar. This is only a temporary variable, it will be removed and replaced with a proper routing system in another PR.

Reply:
Ack